modules/statics/src/base.ts

```diff
@@ -456,6 +456,14 @@ export const enum UnderlyingAsset {
   'txlm:TST-GBQTIOS3XGHB7LVYGBKQVJGCZ3R4JL5E4CBSWJ5ALIJUHBKS6263644L' = 'txlm:TST-GBQTIOS3XGHB7LVYGBKQVJGCZ3R4JL5E4CBSWJ5ALIJUHBKS6263644L',
 }
 
+/**
+ * This is the curve BitGo signs against with the user, backup and BitGo key.
+ */
+export const enum SupportedKeyCurve {
```

Comment:
Nit: Maybe rename to just `KeyCurve`? If it's here, it's assumed to be supported.

Reply:
sure